tests/datadriven/test_variants.py

```diff
@@ -56,11 +57,77 @@ def getDataModelClass(self):
     def getProtocolClass(self):
         return protocol.GAVariantSet
 
-    def verifyVariantsEqual(self, gaVariants, pyvcfVariants):
+    def _compareTwoFloats(self, a, b):
+        afloat = float(a)
+        bfloat = float(b)
+        return (abs(afloat - bfloat) <= 0.001 * abs(bfloat))
+
+    def _compareTwoListFloats(self, a, b):
```

Comment:
Hi @dcolligan @jeromekelleher and @macieksmuga I am actually not sure about using the assertAlmostEqual from the unittest package, which check for round(a-b, 7) == 0, see https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertAlmostEqual, 

for example, this test fails when a and b small, 
```python
>>> a=0.00000005
>>> b=0.00000009
>>> round(a-b, 7) == 0
True
>>> abs(a - b) <= 0.001 * abs(b)
False
```
I am not sure, 0.001 was the best choice here. we could certainly increase the number of digits. but I am not sure using assertAlmostEqual was the best thing? maybe we should just have our own implementation of assertAlmostEqual ( as what I did )? but increase the precision? here is what i propose,

```python
def _assertAlmostEqual(self, a, b, precision=3):
    afloat = float(a)
    bfloat = float(b)
    return (abs(afloat - bfloat) <= 10**-precision * abs(bfloat))
```
what do you think?